### PR TITLE
Feat/trusted hosts

### PR DESCRIPTION
**Pull request by alexloetscher95**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -17,8 +17,7 @@
 APP_ENV=dev
 
 #TRUSTED_PROXIES=127.0.0.0/8,10.0.0.0/8,172.16.0.0/12,192.168.0.0/16
-#TRUSTED_HOSTS='^(localhost|example\.com)$'
-TRUSTED_HOSTS='^.*$'
+TRUSTED_HOSTS='^.*dev\.local$'
```
Comment (aarongerig) [inline .env:20]:
```suggestion
TRUSTED_HOSTS='^.*\.dev\.local$'
```

**Review by aarongerig — CHANGES_REQUESTED** [review]
I would explicitly check for `*.dev.local`, other than that it's all good.

**Review by aarongerig — APPROVED** [review]

File: .env [hunk 2]
```diff
@@ -17,7 +17,7 @@
 APP_ENV=dev
 
 #TRUSTED_PROXIES=127.0.0.0/8,10.0.0.0/8,172.16.0.0/12,192.168.0.0/16
-#TRUSTED_HOSTS='^(localhost|example\.com)$'
+TRUSTED_HOSTS='^.*\.dev\.local$'
```
Comment (aarongerig) [inline .env:20]: What do you think about checking for `*.projects.w-vision.ch` as well?
```suggestion
TRUSTED_HOSTS='(.*\.dev\.local|.*\.projects\.w-vision\.ch)$'
```

**Review by aarongerig — COMMENTED** [review]
@alexloetscher95 One last question...

File: .env (hunk 2 above)
Comment (alexloetscher95) [reply .env]: @aarongerig changed it to
`TRUSTED_HOSTS='^.*(\.dev\.local|\.projects\.w-vision\.ch)$'`
and tested it.
Fine for you?

Comment (aarongerig) [reply .env]: @alexloetscher95 Yes, even better than my suggestion! You can go ahead and merge it.

**Pull request merged**